apps/judicial-system/backend/src/app/formatters/requestPdf.ts

```diff
@@ -159,6 +159,20 @@ function constructRequestPdf(existingCase: Case) {
       paragraphGap: 0,
     })
     .text(' ')
+  if (existingCase.requestProsecutorOnlySession) {
+    doc
+      .font('Helvetica-Bold')
+      .fontSize(14)
+      .lineGap(8)
+      .text('Beiðni um dómþing að varnaraðila fjarstöddum')
+      .font('Helvetica')
+      .fontSize(12)
+      .text(existingCase.prosecutorOnlySessionRequest, {
+        lineGap: 6,
+        paragraphGap: 0,
+      })
+  }
+  doc
```

Comment:
```suggestion
  
  if (existingCase.requestProsecutorOnlySession) {
    doc
      .font('Helvetica-Bold')
      .fontSize(14)
      .lineGap(8)
      .text('Beiðni um dómþing að varnaraðila fjarstöddum')
      .font('Helvetica')
      .fontSize(12)
      .text(existingCase.prosecutorOnlySessionRequest, {
        lineGap: 6,
        paragraphGap: 0,
      })
      .text(' ')
  }
  
  doc
```

Líklega best að rendera eina auka línu á eftir textanum sem þú bættir við.